manifest-list.md

```diff
@@ -64,6 +66,13 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
         This OPTIONAL property specifies an array of strings, each specifying a mandatory CPU feature (for example `sse4` or `aes`).
 
+- **`elements`** *array of objects*
+
+  This OPTIONAL property contains a list of non-manifest objects.
```

Comment:
In the “using the object as an index” use case, the entries here are *likely* to be manifests and manifest lists.  In fact, folks using a manifest-list to switch between platforms could have an index where multiple `elements` (or whatever we end up calling them) have the same `org.opencontainers.ref.name` but differ in other metadata.  That would shorten index → manifest-list → manifest to index → manifest.  So I'd recommend something more generic, like:

> This OPTIONAL property contains an array of descriptors.  No restrictions on the referenced content are implied.